src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        return cinemaHallDao.get(id).orElseThrow(
+                () -> new NoSuchElementException("Can`t get the cinema hall")
```

Comment:
Expand your error message with id.

Reply:
It's better to inform user that `cinema hall by id not found`. Message with that construction is better for user because it explains user that we can find a cinema hall, but the cinema hall with that id does not exist.

Reply:
Make sure to do it everywhere.